pkg/docker/util.js

```diff
@@ -595,6 +595,63 @@
         return deferred.promise();
     };
 
+    util.delete_image_confirm = function confirm(client, image) {
+        var deferred = $.Deferred();
```

Comment:
I know other code here uses this, but to avoid making our future porting live harder: Please use a standard `new Promise` or a `cockpit.defer()`.